channels/testing/websocket.py

```diff
@@ -78,27 +78,33 @@ async def receive_from(self, timeout=1):
         """
         response = await self.receive_output(timeout)
         # Make sure this is a send message
-        assert response["type"] == "websocket.send"
+        assert (
+            response["type"] == "websocket.send"
+        ), f"Expected type 'websocket.send', but was '{response['type']}'"
```

Comment:
This is the useful and important one, the rest I added only for sake of completion.